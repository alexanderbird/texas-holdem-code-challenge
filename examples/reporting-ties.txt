2C 4C 6C 8D
Helli AS 3C
Leese QC 3D
Ariel QD 3H
Mari QH 3S
Darcy TS 5C

---

1 Helli Highcard Ace
4 Leese Highcard Queen (kicker 8 6 4 3)
4 Ariel Highcard Queen (kicker 8 6 4 3)
4 Mari Highcard Queen (kicker 8 6 4 3)
5 Darcy Highcard 10
