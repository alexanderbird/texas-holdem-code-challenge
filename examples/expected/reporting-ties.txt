1 Helli Highcard Ace
4 Leese Highcard Queen
4 Ariel Highcard Queen
4 Mari Highcard Queen
5 Darcy Highcard 10
